minifi-nar-bundles/minifi-provenance-repository-bundle/minifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/MiNiFiPersistentProvenanceRepository.java

```diff
@@ -2065,6 +2065,21 @@ public void run() {
         }
     }
 
+    @Override
+    public String getContainerFileStoreName(final String containerName) {
+        final Map<String, File> map = configuration.getStorageDirectories();
+        final File container = map.get(containerName);
+        if (container == null) {
+            return null;
+        }
+
+        try {
+            return Files.getFileStore(container.toPath()).name();
+        } catch (IOException e) {
+            return null;
```

Comment:
Wondering if there's any use in logging the error?

Reply:
Combing through source, I don't think there is a lot to glean from this scenario.  If I am overlooking something let me know and we should be sure to make the appropriate changes in NiFi, where this is grabbed from; one of the unfortunate idiosyncrasies of the current model that minifi is.

Reply:
I don't disagree -- you would know better than me anyway. :)